Maven app serve fails when executed with default project settings

## Reproduce steps
1. Create a new workspace
2. Enable @jnxplus/nx-boot-maven plugin
3. Generate a new application
4. Try to serve the application `nx serve my-app`

## Result

```
❯ nx serve my-app

 >  NX   Running target serve for project my-app and 1 task(s) it depends on

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

> nx run my-app:build  [local cache]

Executor ran for Build: {}

> nx run my-app:serve

Executor ran for serve: {}
run spring-boot:run
run spring-boot:run ./mvnw spring-boot:run -pl :my-app undefined
[INFO] Scanning for projects...
[INFO]
[INFO] --------------------< com.foo:my-app>--------------------
[INFO] Building my-app 0.0.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.374 s
[INFO] Finished at: 2022-06-14T18:29:14+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Unknown lifecycle phase "undefined". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/LifecyclePhaseNotFoundException

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  NX   Running target "my-app:serve" failed

   Failed tasks:

   - my-app:serve

   Hint: run the command with --verbose for more details.
```

--- a/packages/nx-boot-maven/src/executors/serve/executor.spec.ts
+++ b/packages/nx-boot-maven/src/executors/serve/executor.spec.ts
@@ -32,6 +32,17 @@ describe('Serve Executor', () => {
 
   it('can run', async () => {
     const output = await executor(options, context);
+
     expect(output.success).toBe(true);
+    expect(runCommand).toHaveBeenCalledWith(expect.stringMatching(/spring-boot:run -pl :my-app args$/));
+  });
+
+  describe('when args option is undefined', () => {
+    it('run without extra args', async () => {
+      const output = await executor({} as ServeExecutorSchema, context);
+
+      expect(output.success).toBe(true);
+      expect(runCommand).toHaveBeenCalledWith(expect.stringMatching(/spring-boot:run -pl :my-app$/));
+    });
   });
 });

--- a/packages/nx-boot-maven/src/executors/serve/executor.ts
+++ b/packages/nx-boot-maven/src/executors/serve/executor.ts
@@ -8,8 +8,8 @@ export default async function runExecutor(
 ) {
   logger.info(`Executor ran for serve: ${JSON.stringify(options)}`);
   const result = runCommand(
-    `${getExecutable()} spring-boot:run -pl :${context.projectName} ${
-      options.args
+    `${getExecutable()} spring-boot:run -pl :${context.projectName}${
+      options.args ? ' ' + options.args : ''
     }`
   );
 

